Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM python:3.9-alpine
+
+# Install prereqs
+RUN apk update 
+RUN apk add git gcc
+RUN apk add openssl-dev musl-dev openldap-dev postgresql-dev
+
+# Create directories
+RUN mkdir -p /var/log/fars
+RUN mkdir -p /opt/app/fars
+
+# Copy files
+COPY requirements.txt start-fars.sh /opt/app/
+RUN true
+COPY fars /opt/app/fars
+
+# Set workdir
+WORKDIR /opt/app
+
+RUN pip install --upgrade pip
+RUN pip install wheel
+
+# This is a hack that is required to install python-ldap, as alpine 
+# does not currentl support OpenLDAP 2.5+ which the package uses
+RUN echo -n "INPUT ( libldap.so )" > /usr/lib/libldap_r.so
```

Comment:
begs the question if it's worth using alpine then?

Reply:
Probably not. Changed to an image that actually supports our packages now